chain/bitcoind_conn.go

```diff
@@ -138,7 +180,8 @@ func NewBitcoindConn(cfg *BitcoindConfig) (*BitcoindConn, error) {
 		Pass:                 cfg.Pass,
 		DisableAutoReconnect: false,
 		DisableConnectOnNew:  true,
-		DisableTLS:           true,
+		DisableTLS:           cfg.DisableTLS,
```

Comment:
hmmm actually, seems like the only reason this was added was to get the test in the next commit to pass?

Reply:
ahh fair, let me think about handling that a better way

Reply:
Ok so the issue here is that the rpctest.New() backend created requires TLS, while the client set up in NewBitcoinConn() automatically disables it. Assuming we can't change the DisableTLS option here as it'll break the API -- what if we add an option to rpctest.New() to add the option to DisableTLS. DisableTLS is automatically set to false if we add the option, so I don't think it would break anything in the same way as adding it to NewBitcoinConn. If this seems ok, happy to submit another small PR to the btcd project with this change

Reply:
awesome! yeah that sounds like a good idea :)